### ISSUE TITLE
Inviting Member from connect

### ISSUE THREAD

**Issue by SathyaJayabal** [topcoder-platform/taas-app#191]
1. invite a member from connect to a taas team
2. login as invited member to taas app
3. invited user sees the taas team on the My Teams page even though they havnt not accepted and joined the team yet.
4. Clicking on the Taas team the error "You do not have permissions to perform this action" is displyed.
5. user is able to access team details page after accepting the invitation.

Expected: The taas team must be visible to the invitee only after the user has accepted the invitation.

cc @nkumar-topcoder @maxceem

**Comment by maxceem** [comment]
Technically it would easier if we show projects where user is invited, as API which we use returns them.
What we can do here is:
- show some icon/text to indicate that user is invited to the team
- when user clicks such a team we should redirect to Connect where user could accept invitation
- the only drawback, we cannot make user automatically come back from Connect after accepting invitation, so they would have to manually click link to navigate back to TaaS App

We can also implement accepting invitations from within TaaS UI, but that's quite a big and complex task.

@SathyaJayabal should we reach to Will regarding this?

**Comment by SathyaJayabal** [comment]
@maxceem , how about not showing  the team at all unless they have accepted the invitation in connect as technically, they are not part of the  team unless they have accepted the invite.
